README.md

```
@@ -16,38 +18,71 @@ It's impossible to talk about *"mockingbird"* without remembering the song [Mock
+
+You will also need to start a project with shadow-cljs and install the npm deps.
 
-- Install node dependencies with a simple command: 
+npm:
 
 ``` sh
-npm i
-```
 
-- configure your shadow cljs:
+$ npx create-cljs-project your-project
```

Markdownlint

Dollar signs used before commands without showing output

(MD014, commands-show-output)